app/app.js

```diff
@@ -151,16 +151,21 @@ angular.module('app',
             { map: {}, site: {}, features: {} };
     }])
     // inject the router instance into a `run` block by name
-    // .run(['$uiRouter', '$trace', '$location', function ($uiRouter, $trace, $location) {
+    //.run(['$uiRouter', '$trace', '$location', function ($uiRouter, $trace, $location) {
     //     // * uncomment this to enable the visualizer *
     //     let Visualizer = require('@uirouter/visualizer').Visualizer;
     //     let pluginInstance = $uiRouter.plugin(Visualizer);
     //     $trace.enable('TRANSITION');
     // }])
-    .run(['$rootScope', 'LoadingProgress', function ($rootScope, LoadingProgress) {
+    .run(['$rootScope', 'LoadingProgress', '$transitions', function ($rootScope, LoadingProgress, $transitions) {
         $rootScope.$on('$stateChangeError', console.log.bind(console));
         // this handles the loading-state app-wide
         LoadingProgress.watchTransitions();
+        if (window.ushahidi.gaEnabled) {
+            $transitions.onSuccess({}, function (transition) {
+                window.ga('send', 'pageview', transition.to().name);
```

Comment:
That makes sense. Does transition.to().name translate to a URL though? or just a state name?
We should try to make it a URL since thats expected by GA

Reply:
Good point. I'll change it.

Reply:
I looks like we have two options, `transition.to().name` and `transition.to().url`.

In the GA content list, the `name` format produces the URLs separated by dots. The `url` format produces a URL with slashes. Using `name` shows more deployment-specific info, including record IDs). Both are shown in the example image here, since I switched them out in the middle of collecting data:

<img width="696" alt="screen shot 2018-04-03 at 10 57 10 am" src="https://user-images.githubusercontent.com/6600708/38260761-10ed93f6-372e-11e8-888d-8f5c796aad27.png">